src/main/java/org/cryptomator/ui/mainwindow/MainWindowController.java

```diff
@@ -1,5 +1,15 @@
 package org.cryptomator.ui.mainwindow;
 
+import javafx.beans.Observable;
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.ReadOnlyObjectProperty;
```

Comment:
I wonder why the imports are shuffled? Do you use Project scheme for the code style? In IntelliJ you find this setting in Settings -> Editor -> Code Style -> Java. 

It would be good if you can revert those changes, it keeps the diff small and on the next reformat of someone else it wont't get changed again. (This applies also for the other files)